bundle.js

```diff
@@ -0,0 +1,133 @@
+(() => {
+  var __getOwnPropNames = Object.getOwnPropertyNames;
+  var __commonJS = (cb, mod) => function __require() {
+    return mod || (0, cb[__getOwnPropNames(cb)[0]])((mod = { exports: {} }).exports, mod), mod.exports;
+  };
+
+  // src/newsApi.js
+  var require_newsApi = __commonJS({
+    "src/newsApi.js"(exports, module) {
+      var newsApi2 = class {
+        constructor() {
+          this.url = "https://content.guardianapis.com/search?q=Spotlight&query-fields=headline&show-fields=thumbnail,headline,byline&order-by=newest&api-key=21831b4e-69fe-49f1-a75d-d24709168ad2";
```

Comment:
I would recommend ignoring the bundle.js file because it's showing your api key.